### ISSUE TITLE
Error executing installation command

### ISSUE THREAD

**Issue by RSY96** [googleads/google-ads-php#881]
Good afternoon, I am trying to install the library with the "pecl install grpc" command and it is returning the following error: "Fatal error: Array and string offset access syntax with curly braces is no longer supported in C:\xampp\php\pear \PEAR\Config.php on line 2095". I don't know why but it gives a PHP related syntax error. 

The PHP version I'm using is:
"PHP 8.2.0 (cli) (built: Dec  6 2022 15:31:23) (ZTS Visual C++ 2019 x64)
Copyright (c) The PHP Group
Zend Engine v4.2.0, Copyright (c) Zend Technologies"

And it is not only that file but many more with which it gives a syntax error.

Any solution or idea on how to fix it or install it properly?

Thank you.

**Comment by fiboknacky** [comment]
It's a problem of your Xampp probably. The workaround would be modifying the line of the file (2095) that has this problem. [This post](https://community.apachefriends.org/viewtopic.php?t=81268&p=274260#p274874) has information about this. Could you please try following the workaround there?

**Comment by RSY96** [comment]
Hello,

I've checked the link you sent, thanks for the help but it still doesn't work. As I wrote in the initial post, it is not just one file where the syntax error occurs, but dozens of files.

I tried reinstalling PHP and Xampp and it still shows the same error. And I'm afraid that modifying those files will result in the project not working on the live server with the unmodified php/pear version.

Maybe the library is not compatible and is deprecated?

Thanks greetings.
